docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc

```diff
@@ -47,3 +47,29 @@ com.company.certificate.MyCertificateValidator
 ----
 
 NOTE: You can @Inject CDI and EJB beans in your validator implementation, however since the validator itself is not a Jakarta EE bean, `@PostConstruct` and `@PreDestroy`-annotated methods will not be called on the validator instance.
+
+[[client-certificate-expiration-validator]]
+== Certificate Expiration Validation
+By default, certificate expiration validation is enabled. This validation checks if the certificates in the trust store are valid time wise. If the client certificate has expired or is not valid yet, a warning will be printed in the server logs.
+
+=== Using the Web Admin Console
```

Comment:
Is that the term we use?
I thought we just called it the Admin Console

Reply:
This is a copy paste error from certificate-realm-principal-name.adoc which also has "Web Admin Console". It also has asadmin commands and admin console the wrong way round based on the style guide so that may want changing for documentation consistency.